core/src/conversationProcessor.ts

```diff
@@ -59,6 +68,30 @@ export class ConversationProcessor extends EventEmitter {
     });
   }
 
+  private availableActions(): Action[] {
+    const rambleAction: Action = {
+      name: "rambleAfterResponding",
+      description:
+        "If you want to continue talking, without waiting for a response. Use YES or NO as input.",
+      execute: (input: string, _soul, _conversation) => {
+        devLog(`executing ramble action with input: ${input}`);
+        if (input.toLowerCase() === "no") {
+          return;
+        }
+        this.generatedThoughts.push(
+          new Thought({
+            role: "assistant",
+            entity: this.blueprint.name,
+            action: "RAMBLE",
+            content: "I want to ramble before they respond",
+          })
+        );
+        this.continueThinking();
```

Comment:
Am I misunderstanding or wouldn't this be more clear?

```suggestion
        conversation.generatedThoughts.push(
          new Thought({
            role: "assistant",
            entity: this.blueprint.name,
            action: "RAMBLE",
            content: "I want to ramble before they respond",
          })
        );
        conversation.continueThinking();
```

Reply:
ah good idea. made that change.